Japx/Classes/ObjC/JapxObjC.swift

```diff
@@ -7,7 +7,27 @@
 
 import Foundation
 
-@objc public class JAPXDecoder: NSObject {
+/// Objc bride for `JapxDecodingOptions`
+@objc
+public class JAPXDecodingOptions: NSObject {
+    
+    // MARK: - Private options reference
+    
+    fileprivate var options: JapxDecodingOptions = .default
+    
+    // MARK: - Propery forwarding
+    
+    /// Objc bridge for [`JAPXDecodingOptions.parseNotIncludedRelationships`](x-source-tag://parseNotIncludedRelationships)
+    @objc
+    public var parseNotIncludedRelationships: Bool {
+        get { options.parseNotIncludedRelationships }
+        set { options.parseNotIncludedRelationships = newValue }
+    }
+}
+
+@objc
```

Comment:
```suggestion
@objcMembers
```
and then you can remove all `@objc` modifiers from each method and property.

Reply:
Nice 👍